c2corg_api/models/association.py

```diff
@@ -163,6 +163,7 @@ def _limit_waypoint_fields(query):
 def get_linked_waypoint_parents(document):
     return _limit_waypoint_fields(
         DBSession.query(Waypoint).
+        options(joinedload(getattr(Waypoint, 'geometry'))).
```

Comment:
If only the default geometry is needed (and not `geom_detail`), you should use `loadonly`, e.g. like here: https://github.com/c2corg/v6_api/blob/dcb10e7b0a2b233332be17c26cf84d6b55e38e36/c2corg_api/models/association.py#L158-L160